.circleci/config.yml

```diff
@@ -7,12 +7,14 @@ jobs:
     steps:
       - checkout
       # Build documents
-      - run: sudo pip install -r doc.requirements.txt && cd docs && make html
+      - run:
+          command: |
+            sudo pip install -r docs/requirements.txt
```

Comment:
Why does the command `sudo pip install -r docs/requirements.txt` need ?
`make html` include pip install, I think.

Reply:
I thought too, but circleci/python requires `sudo` to install pip packages in /usr/local

Reply:
I see 😄 Please write the comment something like `#circleci/python requires sudo to install pip packages`.

Reply:
I added comments